pkg/apis/apps/v1alpha1/servicebindingrequest_types.go

```diff
@@ -80,6 +80,58 @@ type ApplicationSelector struct {
 	LabelSelector               *metav1.LabelSelector `json:"labelSelector,omitempty"`
 	metav1.GroupVersionResource `json:",inline"`
 	ResourceRef                 string `json:"resourceRef,omitempty"`
+
+	// BindingPath refers to the path in the application workload's schema
+	// where the binding workload would be referenced.
+	// +optional
+	BindingPath *BindingPath `json:"bindingPath,omitempty"`
+}
+
+var (
```

Comment:
I changed this based on a previous comment. https://github.com/redhat-developer/service-binding-operator/pull/513#discussion_r442963666
It can't be private as these variables are used from other packages.

Reply:
Changed to constant. 1cf2567c50738891dcfcf60f0872ad5718058e74